music_assistant/providers/sonos_s1/__init__.py

```diff
@@ -362,6 +377,25 @@ async def poll_player(self, player_id: str) -> None:
 
     async def discover_players(self) -> None:
         """Discover Sonos players on the network."""
+        manual_ip_config: str | None
+        if (manual_ip_config := self.config.get_value(CONF_IPS)) is not None:  # type: ignore[assignment]
+            ips = set(map(str.strip, manual_ip_config.split(",")))
+            for ip in ips:
+                try:
+                    player = SoCo(ip)
+                    self._add_player(player)
+                except RequestException as err:
+                    # player is offline
+                    self.logger.debug("Failed to add SonosPlayer %s: %s", player, err)
+                except Exception as err:
+                    self.logger.warning(
+                        "Failed to add SonosPlayer %s: %s",
+                        player,
+                        err,
+                        exc_info=err if self.logger.isEnabledFor(10) else None,
+                    )
+            return
```

Comment:
why the return ?
imo we should make the manual IP's option an ADDITIONAL discovery method, not completely disabling the primary discovery method.